cmd/accounts.go

```diff
@@ -0,0 +1,39 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// accountsCmd represents the accounts command
+var accountsCmd = &cobra.Command{
+	Use:   "accounts",
+	Short: "Exports the account data.",
+	Long:  `Exports historical account data within the specified ledger range to an output file.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		/*
+			Functionality planning:
+			1. Read in start and end ledger numbers
+			2. Provide ledger number to ingestion system and receive data
+			3. Convert data from ingestion into an array of Ledger structs with length equal to the limit
+			4. Write array to output file
+			TODO: Consider having a way to export directly to BigQuery using bigquery library
+		*/
+	},
+}
+
+func init() {
+	rootCmd.AddCommand(accountsCmd)
+	/*
+		Needed flags:
+			TODO: determine if providing ledger sequence number or timestamp is preferable (possibly could do both)
+
+			start-time: the time for the beginning of the period to export; default to genesis ledger's creation time
```

Comment:
This is an interesting default. I'm not sure I agree with it, since a user could easily accidentally kick it off from the genesis ledger, rather than just the last five minutes. I don't think we need to change this comment yet, because that might be the right use case. This will become clearer in the next few weeks as we implement more.